.travis.yml

```diff
@@ -13,4 +13,9 @@ jdk:
 
 script:
   - curl https://raw.githubusercontent.com/scala-native/scala-native/b7851c4dc2a22c9fd9202aea3d97e2a68910b918/scripts/travis_setup.sh | bash -x
+  # TODO: remove these later
+  - git clone --depth 1 https://github.com/lihaoyi/sourcecode.git
+  - cd sourcecode && ./mill sourcecode.jvm[0.24.0-RC1].publishLocal && cd ..
```

Comment:
I believe sourcecode and utest are already available on Maven for these versions. So no need to publish locally.